### DISCUSSION TITLE
How to install older version?

### DISCUSSION THREAD

**Discussion by gagecarto** [developmentseed/titiler#1000] [Q&A] [answered]
I just recently installed TiTiler for a new project and am getting some errors. I read about breaking changes in the new version so I am guessing that is why. I installed like below
`python -m pip  install titiler.application (also installs core, extensions and mosaic)`

How would I install an older version which does not include some of the breaking changes?

**Comment by vincentsarago** [comment, marked as answer]
> How would I install an older version which does not include some of the breaking changes?

 titiler.application have `titiler.*` dependencies pinned to specific version 
https://github.com/developmentseed/titiler/blob/1825dce9f8c8a478e7cef3f3172aa5f660e7454a/src/titiler/application/pyproject.toml#L34-L36

so if you install `titiler.application=={desired version}` you should be fine 🤷
